src/components/FavoritesButton/FavoritesButton.tsx

```diff
@@ -21,7 +22,7 @@ const FavoritesButton = ({ cityName }: Props) => {
 
   return (
     <Button data-testid={FavoritesButtonTestIds.Button} active={isFavorite} onClick={handleClick}>
-      <Heart size={13} />
+      <Heart size={size} />
```

Comment:
**Suggestion** What do you think about adding the size to some kind of theme?

Reply:
I don't know if this is a good solution, because this value is different for each view

Reply:
Okay